### PR TITLE
creates Kai Landre page

### LINKED ISSUE #8: Project 9: Kai Landre Video
image

### PR DESCRIPTION
**Pull request by sssoja**
### What changes have you made?
- added the video content to the Kai Landre page (using a placeholder url for now)
- installed dependencies for [React Player](https://www.npmjs.com/package/react-player) 

### Which issue(s) does this PR fix?

Fixes #8 

### Screenshots (if there are design changes)
Screenshot 2020-11-03 at 15 16 38

Screenshot 2020-11-03 at 15 16 58

### How to test
- nav to http://localhost:3000/projects/kailandre and check that you can play the video on click
- check that there is no autoplay 
- check that you can pause using the space key
- ensure that the video size is responsive on mobile and desktop
- make sure you can toggle forward and back using mouse keys 
- comment out lines 21 - 23 and comment in line 24 to test the page preview

### CODE REVIEW

File: src/components/project-pages/KaiLandrePreview.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import styled from "styled-components";
+import Timer from "../Timer";
+import { FlexboxProps, flexbox, layout, LayoutProps } from "styled-system";
+import dragon from "../assets/project-page/dragon.png";
+
+const Main = styled.main<FlexboxProps>`
+  display: flex;
+  height: 100%;
+  overflow: hidden;
```
Comment (Jbarget) [inline src/components/project-pages/KaiLandrePreview.tsx:10]: this is unnecessary i think

File: src/components/project-pages/KaiLandre.tsx [hunk 2]
```diff
@@ -1,33 +1,28 @@
...
-  ${flexbox};
-`;
-
-const Img = styled.img<LayoutProps>`
+const Container = styled.div<LayoutProps>`
   ${layout};
 `;
 
 const KaiLandre = () => {
+  const url = 'https://www.youtube.com/watch?v=AGSH_acR4wA'
```
Comment (Jbarget) [inline src/components/project-pages/KaiLandre.tsx:18]: is it definitely a youtube video that you'll be sent? and not Vimeo or something else?

**Review by Jbarget — COMMENTED** [review]
nice work, 

fyi Matts looking into the font loading issue

File: src/components/project-pages/KaiLandre.tsx (hunk 2 above)
Comment (sssoja) [reply src/components/project-pages/KaiLandre.tsx:18]: He said YouTube but [the React player also works for Vimeo](https://cookpete.com/react-player/)

**Review by Jbarget — APPROVED** [review]

**Pull request merged**